cabal.project

```diff
@@ -32,7 +32,7 @@ packages:
   -- doc/cookbook/jwt-and-basic-auth/
   doc/cookbook/pagination
   -- doc/cookbook/sentry
-  doc/cookbook/testing
+--  doc/cookbook/testing
```

Comment:
Why this?

Reply:
Because of `servant-quickcheck` :) I hoped that CI will pas with this commented, but it fails anyway. I'll revert.